### PR TITLE
stats: update statistic docs

### PR DESCRIPTION
**Pull request by fipped**
PTAL @lamxTyler @coocood

### CODE REVIEW

**Comment by shenli** [comment]
@fipped Please sign the CLA.

File: sql/statistics.md [hunk 1]
```diff
@@ -122,3 +126,22 @@ SHOW STATS_BUCKETS [ShowLikeOrWhere]
...
 > 该语句会删除 TableName 中所有的统计信息。
 ```
+
+## 统计信息的导入导出
+
+### 导出统计信息
+
+统计信息的导出接口为：
+```
+http://localhost:10080/stats/dump/${db_name}/${table_name}
```
Comment (shenli) [inline sql/statistics.md]: localhost:10080 -> {tidb-server-ip}:{tidb-server-status-port}

**Comment by coocood** [comment]
LGTM

File: sql/statistics.md [hunk 2]
```diff
@@ -122,3 +126,22 @@ SHOW STATS_BUCKETS [ShowLikeOrWhere]
...
 > 该语句会删除 TableName 中所有的统计信息。
 ```
+
+## 统计信息的导入导出
+
+### 导出统计信息
+
+统计信息的导出接口为：
+```
+http://{tidb-server-ip}:{tidb-server-status-port}/stats/dump/${db_name}/${table_name}
```
Comment (lamxTyler) [inline sql/statistics.md]: Either put the `$` sign at all variables or do not put it anywhere.

File: sql/statistics.md [hunk 3]
```diff
@@ -122,3 +126,22 @@ SHOW STATS_BUCKETS [ShowLikeOrWhere]
...
+
+通过该接口可以获取数据库 `${db_name}` 中的表 `${table_name}` 的 json 格式的统计信息。
+
+### 导入统计信息
+
+语法：
+```sql
+LOAD STATS 'file_name'
+```
+`file_name` 为要导入的统计信息的文件名。该文件一般是通过统计信息导出接口得到的 json 文件。
```
Comment (lamxTyler) [inline sql/statistics.md]: Put it in the SQL statement block like others do.

File: sql/statistics.md [hunk 4]
```diff
@@ -20,6 +20,9 @@ ANALYZE TABLE TableNameList
 
 ANALYZE TABLE TableName INDEX IndexNameList
 > 该语句会收集 TableName 中所有的 IndexNameList 中的索引列的统计信息。
+
+ANALYZE TABLE TableName INDEX
```
Comment (lamxTyler) [inline sql/statistics.md]: You can just combine it with the upper one.

**Pull request merged**